apps/docs/content/components/layout/Grid.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: Grid
-description: The Grid component is used to create a grid container and provides some shortcuts for the grid properties.
+description: A grid component is a component used to create a grid container.
```

Comment:
I'm debating if A grid is the way to go...
"The Grid" is specifically referring to the Grid component that we created, not some random grid component.

Reply:
I'm thinking of going with The grid component is used to ...

Reply:
ok, sounds good to me.